### PR TITLE
Add some API official references

### PR DESCRIPTION
**Pull request by chemelli74**
Added some API official references

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -95,3 +95,9 @@ Credits / Special Thanks
 - https://github.com/aaska
 
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references:
+https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf
+https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/FileStation/All/enu/Synology_File_Station_API_Guide.pdf
+https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/SurveillanceStation/All/enu/Surveillance_Station_Web_API.pdf
+https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf
```
Comment (Quentame) [inline README.rst:103]: Make links a list + name list items like this \`[Service] API documentation\<link\>\`_

Order it alphabetically

File: README.rst [hunk 2]
```diff
@@ -95,3 +95,9 @@ Credits / Special Thanks
 - https://github.com/aaska
 
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references:
```
Comment (Quentame) [inline README.rst:99]:
```suggestion
Official references
===================
```

**Comment by Quentame** [comment]
Thanks for the PR @chemelli74 😉

File: README.rst (hunk 1 above)
Comment (Quentame) [reply README.rst:104]: This is not resolved !

Comment (chemelli74) [reply README.rst]: Sorry, should be fixed now.

File: README.rst [hunk 3]
```diff
@@ -95,3 +95,14 @@ Credits / Special Thanks
 - https://github.com/aaska
 
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references
+===================
+
+- [Calendar] API documentation: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf
```
Comment (Quentame) [inline README.rst:102]:
```suggestion
- `Calendar API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf>`_
```

File: README.rst [hunk 4]
```diff
@@ -95,3 +95,14 @@ Credits / Special Thanks
 - https://github.com/aaska
 
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references
+===================
+
+- [Calendar] API documentation: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf
+
+- [Download Station] API documentation: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf
```
Comment (Quentame) [inline README.rst:104]:
```suggestion
- `Download Station API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf>`_
```

File: README.rst [hunk 5]
```diff
@@ -95,3 +95,14 @@ Credits / Special Thanks
...
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references
+===================
+
+- [Calendar] API documentation: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf
+
+- [Download Station] API documentation: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf
+
+- [File Station] API documentation: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/FileStation/All/enu/Synology_File_Station_API_Guide.pdf
```
Comment (Quentame) [inline README.rst:106]:
```suggestion
- `File Station API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/FileStation/All/enu/Synology_File_Station_API_Guide.pdf>`_
```

File: README.rst [hunk 6]
```diff
@@ -95,3 +95,14 @@ Credits / Special Thanks
...
+
+- [Surveillance STation] API documentation: https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/SurveillanceStation/All/enu/Surveillance_Station_Web_API.pdf
```
Comment (Quentame) [inline README.rst:108]:
```suggestion
- `Surveillance Station API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/SurveillanceStation/All/enu/Surveillance_Station_Web_API.pdf>`_
```

File: README.rst [hunk 7]
```diff
@@ -95,3 +95,14 @@ Credits / Special Thanks
 - https://github.com/aaska
 
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references
+===================
+
+- `Calendar API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf>`_
```
Comment (Quentame) [inline README.rst:102]:
```suggestion
- `Calendar API documentation <https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf>`_
```

File: README.rst [hunk 8]
```diff
@@ -95,3 +95,14 @@ Credits / Special Thanks
 - https://github.com/aaska
 
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references
+===================
+
+- `Calendar API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf>`_
+
+- `Download Station API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf>`_
```
Comment (Quentame) [inline README.rst:104]:
```suggestion
- `Download Station API documentation <https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf>`_
```

File: README.rst [hunk 9]
```diff
@@ -95,3 +95,14 @@ Credits / Special Thanks
...
 Found Synology API "documentation" on this repo : https://github.com/kwent/syno/tree/master/definitions
+
+Official references
+===================
+
+- `Calendar API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/Calendar/2.4/enu/Synology_Calendar_API_Guide_enu.pdf>`_
+
+- `Download Station API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/DownloadStation/All/enu/Synology_Download_Station_Web_API.pdf>`_
+
+- `File Station API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/FileStation/All/enu/Synology_File_Station_API_Guide.pdf>`_
```
Comment (Quentame) [inline README.rst:106]:
```suggestion
- `File Station API documentation <https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/FileStation/All/enu/Synology_File_Station_API_Guide.pdf>`_
```

File: README.rst [hunk 10]
```diff
@@ -95,3 +95,14 @@ Credits / Special Thanks
...
+
+- `Surveillance Station API documentation<https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/SurveillanceStation/All/enu/Surveillance_Station_Web_API.pdf>`_
```
Comment (Quentame) [inline README.rst:108]:
```suggestion
- `Surveillance Station API documentation <https://global.download.synology.com/download/Document/Software/DeveloperGuide/Package/SurveillanceStation/All/enu/Surveillance_Station_Web_API.pdf>`_
```

**Pull request merged**